src/utils/getS3SignedUrl.js

```diff
@@ -0,0 +1,26 @@
+const AWS = require('aws-sdk');
```

Comment:
There's `utils/aws/s3.js`, I would move this file over there instead of having it in `utils`

Reply:
moved 👍